letsencrypt/cli.py

```diff
@@ -44,6 +44,11 @@
 
 logger = logging.getLogger(__name__)
 
+# For help strings, figure out how the user ran us.
+# When invoked from letsencrypt-auto, sys.argv[0] is something like:
+# /home/user/.local/share/letsencrypt/bin/letsencrypt"
+fragment = os.path.join(".local", "share", "letsencrypt")
+cli_command = "letsencrypt-auto" if fragment in sys.argv[0] else "letsencrypt"
```

Comment:
I looked into doing this in a more exhaustive / thorough way, and it's probably impossible.  By default, `sudo` doesn't preserve environment variables (you need to pass in `-E` for that), so we can't tell if the user had `VENV_PATH` or `XDG_DATA_HOME` set.  But I think it's going to be very rare for those to be in play, so as long as we're cautious about what we hang on this hook, it should be okay.  I'll add a comment to that effect...